component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateUtil.java

```diff
@@ -374,4 +376,17 @@ private static boolean isUserExists(String userName, AuthenticationContext authe
         }
     }
 
-}
\ No newline at end of file
+    public static boolean isAccountLock(AuthenticatedUser user) {
```

Comment:
Changes might incur a scenario where the user is `null`. Therefore, we need to handle `null` scenario as well.